easybuild/easyconfigs/b/BLASR/BLASR-5.3.3-gompi-2019a.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MesonNinja'
+
+name = 'BLASR'
+version = '5.3.3'
+
+homepage = 'https://github.com/PacificBiosciences/blasr'
+description = "The PacBio® long read aligner"
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
+toolchainopts = {'cstd': 'c++14'}
+
+source_urls = ['https://github.com/PacificBiosciences/blasr/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['0c69f0ed04c6998fdd60969dc6c87f29298453a230767f5f206ccceca939dc52']
+
+builddependencies = [
+    ('Meson', '0.50.0', '-Python-3.7.2'),
```

Comment:
@boegel even though this is Python3 only, is it ok to **not** have a Python versionsuffix for BLASR?

Reply:
@smoors Yes in this case, because `Meson` is just a build dependency...